x-pack/plugins/timelines/public/index.ts

```diff
@@ -21,6 +21,7 @@ export type {
 } from '../common/search_strategy/common';
 export { Direction } from '../common/search_strategy/common';
 export { tGridReducer } from './store/t_grid/reducer';
+export { LastUpdatedAt } from './components';
```

Comment:
I know this is by far the easiest way to do this, but the timelines bundle is already much bigger than it should be. Would you mind loading this component in security_solution asynchronously? It's not hard to do, see public/plugin.ts and and public/methods/index.ts for examples. and even just adding this 1 component to the top level public/index.ts increases the bundle size by an almost comical 200kb

Reply:
@michaelolo24 's suggestion works as well